packages/cli/src/api/catalog.test.ts

```diff
@@ -282,12 +283,13 @@ describe("Catalog", () => {
         mockConfig()
       )
 
-      mockConsole(async (console) => {
+      expect.assertions(2)
+      await mockConsole(async (console) => {
         const messages = await catalog.collect()
         expect(console.error).toBeCalledWith(
           expect.stringContaining(`Cannot process file`)
         )
-        expect(messages).toMatchSnapshot()
+        expect(messages).toBeFalsy()
```

Comment:
that case what completely broken. `await` was forgotten + issue with `mockConsole`